auditlog/receivers.py

```diff
@@ -52,6 +52,9 @@ def log_update(sender, instance, **kwargs):
     if instance.pk is not None:
         update_fields = kwargs.get("update_fields", None)
         old = sender.objects.filter(pk=instance.pk).first()
+        if not old:
```

Comment:
@hramezani, should I replace this check to `if instance._state.adding`? https://docs.djangoproject.com/en/4.2/ref/models/instances/#django.db.models.Model._state

Reply:
I thought about this for a short while and decided to use the `adding` flag (it exists for a reason).